docs/formats.rst

```diff
@@ -6,7 +6,15 @@ Preferred Extensions and Mime Types
 
 Images can generally be read regardless of their name.  By default, when opening an image with ``large_image.open()``, each tile source reader is tried in turn until one source can open the file.  Each source lists preferred file extensions and mime types with a priority level.  If the file ends with one of these extensions or has one of these mimetypes, the order that the source readers are tried is adjusted based on the specified priority.
 
-The file extensions and mime types that are listed by the core sources that can affect source processing order are listed below.  See ``large_image.listSources()`` for details about priority of the different source and the ``large_image.constants.SourcePriority`` for the priority meaning.
+The file extensions and mime types that are listed by the core sources that can affect source processing order are listed below.  See ``large_image.listSources()`` for details about priority of the different sources and the ``large_image.constants.SourcePriority`` for the priority meaning.
+
+The following table describes the primary formats supported by ``large-image`` and the acceptable extensions for each format. This table also describes the ``large-image-source-*`` modules that can be used to read each format.
```

Comment:
I'd reword this a little.  I'd change "primary" to "some of the" and replace "acceptable" with "common".

My rationale is that we list formats I wouldn't consider primary (GIF and BMP, even LIF).  And, we actually can work with files regardless of extension, and some have multiple common extensions (e.g., .tif, .tiff; .jpeg, .jpg (and even .jpe); .j2k, .jp2, etc.).

Reply:
Updated in [ba08389](https://github.com/girder/large_image/pull/1622/commits/ba08389bb4f16b06377ebbf3bb2d78e1b235416d)